### PR TITLE
Fix ad205

### PR DESCRIPTION
**Pull request by ASayre**
Fixes AdamDriscoll#205

Root cause was an erroneous MEF Import/Export, which would cause race conditions under certain circumstances, which initializing before the MEF Graph had been completed.

@HuanhuanSunMSFT @adamdriscoll

### CODE REVIEW

**Comment by ASayre** [comment]
As an addl TODO: Should we be [Import]ing the IClassificationTypeRegistryService / IClassificationFormatMapService/ITextBufferFactoryService? 

Any particular reason we queried IComponentModel directly?

@adamdriscoll

File: PowerShellTools/Diagnostics/DiagnosticsDialogPage.cs [hunk 1]
```diff
@@ -1,30 +1,41 @@
...
+﻿using System;
+using System.ComponentModel;
 using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.Shell;
 
 namespace PowerShellTools.Diagnostics
 {
-    class DiagnosticsDialogPage : DialogPage
+    /// <summary>
+    /// This class contains the DialogPage for all dianostic related items.
```
Comment (HuanhuanSunMSFT) [inline PowerShellTools/Diagnostics/DiagnosticsDialogPage.cs]: nit: typo diagnostic

**Comment by HuanhuanSunMSFT** [comment]
Looks good.

**Comment by ASayre** [comment]
Fixed typo!

**Pull request merged**